src/App.js

```diff
@@ -81,8 +86,20 @@ if (__DEV__) {
     monitorNetwork(showNetworkRequests, showNetworkResponses);
 } else {
   // eslint-disable-next-line no-inner-declarations
-  async function initSentry() {
+  async function initSentryAndCheckForFedoraMode() {
```

Comment:
can we fail more nicely if an error occurs inside this function trying to get current package etc / update metadata